### PR TITLE
docs: update README

### LINKED ISSUE #4350: [Docs]: Update the README
### Is there an existing issue already for your request/idea?

- [X] I have searched for an existing issue, and could not find anything. I believe this is a new documentation enhancement to be evaluated.

### What problem in the existing documentation this issue aims to solve?

The README file is outdated.

### Describe what additions need to be done to the documentation

Change the README file

### Describe what pages need to change in the documentation, if any

_No response_

### Describe what pages need to be removed from the documentation, if any

_No response_

### Additional context

_No response_

### Backport?

No

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by gbartolini**
Closes #4350

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,16 +122,11 @@ MariaDB cluster).
 - [FAQ](docs/src/faq.md)
 - [Blog](https://cloudnative-pg.io/blog/)
 
-### Useful links
+## Adopters
 
-- [Data on Kubernetes (DoK) Community](https://dok.community/)
-- ["Recommended Architectures for PostgreSQL in Kubernetes" by Gabriele Bartolini](https://www.cncf.io/blog/2023/09/29/recommended-architectures-for-postgresql-in-kubernetes/)
-- ["How CloudNativePG manages physical replication slots for PostgreSQL in Kubernetes" by Gabriele Bartolini](https://www.enterprisedb.com/blog/how-cloudnativepg-manages-replication-slots) (September 2023)
```
Comment (mnencia) [inline README.md:124]: This line has not been moved together with the others. Is it intentional?

**Review by mnencia — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (gbartolini) [reply README.md:124]: Yes. IMHO it is about a very specific topic for the main README.

File: README.md [hunk 2]
```diff
@@ -17,11 +19,12 @@ CloudNativePG was originally built and sponsored by [EDB](https://www.enterprise
 - [Code of conduct](CODE_OF_CONDUCT.md)
 - [Governance policies](GOVERNANCE.md)
 - [Contributing](CONTRIBUTING.md)
+- [Adopters](ADOPTERS.md)
 - [License](LICENSE)
 
 ## Getting Started
 
-The best way to get started is with the ["Quickstart"](docs/src/quickstart.md)
+The best way to get started is with the ["Quickstart"](https://cloudnative-pg.io/documentation/current/quickstart/)
```
Comment (jsilvela) [inline README.md:27]: I think it's better to keep links local. Both in GH and in local checkouts, does what we want, and the link could mask mistakes.

File: README.md [hunk 3]
```diff
@@ -52,8 +55,8 @@ managing complex applications.
 
 As a result, the operator is responsible for managing the status of the
 `Cluster` resource, keeping it up to date with the information that each
-PostgreSQL instance manager regularly reports back through the API server. Such
-changes might trigger, for example, actions like:
+PostgreSQL instance manager regularly reports back through the API server.
+Such changes might trigger, for example, actions like:
```
Comment (jsilvela) [inline README.md:59]: "such changes" seems to be a reference to a previous mention of changes.
```suggestion
Changes to the cluster status might trigger, for example, actions like:
```

File: README.md [hunk 4]
```diff
@@ -86,12 +89,14 @@ intermediate management tool responsible for high availability and failover
...
 CloudNativePG also manages additional resources to help the `Cluster` resource
-manage PostgreSQL - currently `ScheduledBackup`, `Pooler`, and `Backup`.
+manage PostgreSQL - currently `Backup`, `ClusterImageCatalog`, `ImageCatalog`,
+`Pooler`, and `ScheduledBackup`.
 
-Fully embracing Kubernetes also means that, in case of failure of the whole
-Kubernetes cluster, the operator won’t do anything, postponing any decision to
-when the cluster is back up again. In the meantime, Postgres instances should
-continue running according to the last known state of the cluster.
+Fully embracing Kubernetes involves a hands-off approach during temporary
```
Comment (jsilvela) [inline README.md:95]: Not loving "involves" here. I think the previous version with "means" is better.
```suggestion
Fully embracing Kubernetes means adopting a hands-off approach during temporary
```

File: README.md [hunk 5]
```diff
@@ -138,15 +138,28 @@ The current maintainers of the CloudNativePG project are:
 - Jonathan Gonzalez (EDB)
 - Marco Nenciarini (EDB)
 - Armando Ruocco (EDB)
-- Philippe Scorsolini (upbound)
+- Philippe Scorsolini (Upbound)
 
 They are listed in the [CODEOWNERS](CODEOWNERS) file.
 
-## Adopters
+### CloudNativePG at KubeCon
```
Comment (jsilvela) [inline README.md:145]: Nice. I also think we need to add this to the website (PR coming soon for that repo)

**Review by jsilvela — APPROVED** [review]
A few optional comments.

File: README.md (hunk 2 above)
Comment (gbartolini) [reply README.md:27]: Ok!

**Pull request merged**